spec/controllers/reductions_controller_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'spec_helper'
+require 'ostruct'
+
+describe ReductionsController, :type => :controller do
+  let(:reductions) {
+    [
+      Reduction.create!(workflow_id: 1234, subject_id: 5678, reducer_key: 'r', data: '1'),
+      Reduction.create!(workflow_id: 1234, subject_id: 5678, reducer_key: 's', data: '2'),
+      Reduction.create!(workflow_id: 1234, subject_id: 6789, reducer_key: 'r', data: '3')
+    ]
+  }
+
+  def tamper_auth
+    # i can do what i want
+    #
+    # --ron swanson
+    allow_any_instance_of(ReductionsController).to receive(:authenticated?).and_return(true)
+    # allow_any_instance_of(ReductionsController).to receive(:authorized?).and_return(true)
+    allow_any_instance_of(Credential).to receive(:logged_in?).and_return(true)
+    allow_any_instance_of(Credential).to receive(:expired?).and_return(false)
+    allow_any_instance_of(Credential).to receive(:admin?).and_return(true)
+  end
+
+  before do
+    Subject.create! id: 5678
+    Subject.create! id: 6789
+    Workflow.create! id: 1234, project_id: 12
+  end
+
+  after do
```

Comment:
No need to do this. RSpec is configured to run each test in a database transaction, and issue a rollback (this is all through the `database_cleaner` gem).